src/networking.c

```diff
@@ -2196,7 +2196,8 @@ void readQueryFromClient(connection *conn) {
     c->lastinteraction = server.unixtime;
     if (c->flags & CLIENT_MASTER) c->read_reploff += nread;
     atomicIncr(server.stat_net_input_bytes, nread);
-    if (sdslen(c->querybuf) > server.client_max_querybuf_len) {
+    // For non-replication client, check the command length with query buffer limit
```

Comment:
nitpick, few issues with this comment:
1. uses a line comment rather than `/*`
2. mentions "command length" (this limit isn't about commands, and not even about arguments).
3. i'm not at all sure any comment is needed, same as before, the code seems already clear.

so it could just be something like
```c
/* Fail if the query buffer reached its limit (excluding master link) */
```
but i'm not sure it's needed.

Reply:
I can remove this comment line. I don’t think it adds much value either